Palette Go SDK (#3577)
spectrocloud/librarium: docs/docs-content/automation/automation.md

  [self-hosted Palette](../enterprise-version/enterprise-version.md) instance and deploying a
  [Private Cloud Gateway](../clusters/pcg/pcg.md).

> - Palette Go SDK - Provides client configuration through functional options and boilerplate functions, enabling Go

Reviewer: ```suggestion
- Palette Go SDK - Provides client configuration through a set of public libraries, enabling Go
```

> - Palette Go SDK - Provides client configuration through a set of public libraries, enabling Go developers to interact

Reviewer 2: suggestion:

Palette Go SDK - Enables developers to interact with Palette APIs for automated resource management using Go.
